test/specs/utils/tablify.js

```diff
@@ -66,6 +66,18 @@ describe('tablify util', () => {
 			);
 		});
 	});
+	Given('a cyclic object', given.aCyclicObject, () => {
+		When('the object is tablified', when.theObjectIsTablified, () => {
+			Then(
+				'the returned table should have a head with the cyclic object’s keys',
+				then.theReturnedTableShouldHaveAHeadWithTheCyclicObjectKeys,
+			);
+			Then(
+				'the returned table should have a row with the cyclic object’s values including an error for the cyclic value',
```

Comment:
`including a message for the cycle value` (and corresponding change for the function name).